examples/wkwebview/wkwebview/LinkViewController.swift

```diff
@@ -69,62 +71,81 @@ class LinkViewController: UIViewController, WKNavigationDelegate {
     }
     
     func webView(_ webView: WKWebView,
-                 decidePolicyForNavigationAction navigationAction: WKNavigationAction,
-                 decisionHandler: ((WKNavigationActionPolicy) -> Void)) {
+                 decidePolicyFor navigationAction: WKNavigationAction,
+                 decisionHandler: @escaping ((WKNavigationActionPolicy) -> Void)) {
```

Comment:
I struggled with this decision -- I eventually decided to just convert it to Swift3, no worries if you disagree :smile: